lib/data_hygiene/document_status_checker.rb

```diff
@@ -0,0 +1,40 @@
+class DataHygiene::DocumentStatusChecker
+  def initialize(document)
+    @document = document
+  end
+
+  def content_store?
+    routes.each do |route|
+      content_item = GdsApi.content_store.content_item(route[:path])
+      updated_at = DateTime.parse(content_item["updated_at"])
+      return false unless updated_at > edition.published_at # content-store must be later due to latency
+    rescue GdsApi::ContentStore::ItemNotFound
+      return false
+    end
+
+    true
+  end
+
+  def router?
+    routes.each do |route|
+      route = GdsApi.router.get_route(base_path)
+      return false unless route["backend_id"] == edition.rendering_app
```

Comment:
This might just be some divergent terminology, but is it correct that `backend_id` would contain the `rendering_app` and not the publishing app?

Reply:
Yeah, basically `backend_id` is the name of the app that router will send requests to, so it has to be set to the frontend app that will handle the request.